pkgs/by-name/pi/piano-rs/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeWrapper,
+  pkg-config,
+  python310,
+  alsa-lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "piano-rs";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ritiek";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-qZeH9xXQPIOJ87mvLahnJB3DuEgLX0EAXPvECgxNlq0=";
+  };
+
+  cargoHash = "sha256-vDqfWXeQVEnMWMjhAG/A0afff7dWMoQejDZjcVlYBMQ=";
+
+  nativeBuildInputs = [
+    makeWrapper
+    pkg-config
+  ];
+
+  buildInputs = [
+    python310
```

Comment:
```suggestion
```

Reply:
Hi! Interesting. I remember having to specify `python310` so I could get the package to build. Otherwise it used to complain about rustbox (which in turn depends on termbox) requiring python to build.

I tried building it without `python310` now and it indeed seems to building fine, weird. Thanks for the review!